backend/services/container_migration_service.py

```
@@ -6,7 +6,10 @@
     TransferMethod, HostInfo, IdentifierType
 )
 from ..docker_ops import DockerOperations, ContainerInfo, NetworkInfo
-from ..zfs_ops import ZFSOperations
+from ..zfs_operations.factories.service_factory import create_default_service_factory
+from ..zfs_operations.services.dataset_service import DatasetService
+from ..zfs_operations.services.snapshot_service import SnapshotService as NewSnapshotService
+from ..zfs_operations.core.value_objects.dataset_name import DatasetName
```

Ruff (0.11.9)

`..zfs_operations.core.value_objects.dataset_name.DatasetName` imported but unused

Remove unused import: `..zfs_operations.core.value_objects.dataset_name.DatasetName`

(F401)